homeassistant/components/sensor/github.py

```diff
@@ -0,0 +1,294 @@
+"""
+Support for GitHub.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.github/
+"""
+from datetime import timedelta
+import logging
+import json
+import voluptuous as vol
```

Comment:
Please keep import grouped and ordered. If you don't want to do it manually, use `isort`.

Reply:
I *think* I managed to sort the imports. Not sat at my usual pc so I had to do a manual sort